Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -1421,6 +1458,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         await self.tunnelFailureMonitor.stop()
         await self.latencyMonitor.stop()
         await self.entitlementMonitor.stop()
+        await self.serverStatusMonitor.stop()
```

Comment:
Should we also stop this while `sleep()`?

Reply:
Yep, thanks for the reminder, I overlooked that we don't call this function when sleeping. Perhaps we should 🤔